fdbserver/storageserver.actor.cpp

```diff
@@ -536,7 +587,12 @@ struct StorageServer {
 	} counters;
 
 	StorageServer(IKeyValueStore* storage, Reference<AsyncVar<ServerDBInfo>> const& db, StorageServerInterface const& ssi)
-		:	instanceID(deterministicRandom()->randomUniqueID().first()),
+		:	fetchKeysHistograms(
+				Histogram::getHistogram(STORAGESERVER_HISTOGRAM_GROUP, FETCH_KEYS_LATENCY_HISTOGRAM, Histogram::Unit::microseconds),
+				Histogram::getHistogram(STORAGESERVER_HISTOGRAM_GROUP, FETCH_KEYS_BYTES_HISTOGRAM, Histogram::Unit::bytes),
+				Histogram::getHistogram(STORAGESERVER_HISTOGRAM_GROUP, FETCH_KEYS_BYTES_PER_SECOND_HISTOGRAM, Histogram::Unit::bytes)
```

Comment:
Either make three fields (`fetchKeysLatency`...), or move these constructor calls to FetchKeysHistograms(), which would then take zero parameters.

As it is, someone could reorder two of these lines, and it would lead to incorrect metrics.  It would be hard to spot because the constructor definition is far away, and all three of these parameters have the same type.